static-build/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.1)
```

Comment:
I found an old chat message, where @igormunkin said that 45d97c41da35384be0dac70721e5381618e2c7f8 leads to a build fail due to lack of `-Wno-gnu-alignof-expression` on Mac OS (at least in the static build). Don't know details.

Here we have the green CI, so everything should be fine.